src/journal/journald-server.c

```diff
@@ -78,8 +78,8 @@
 #define USER_JOURNALS_MAX 1024
 
 #define DEFAULT_SYNC_INTERVAL_USEC (5*USEC_PER_MINUTE)
-#define DEFAULT_RATE_LIMIT_INTERVAL (30*USEC_PER_SEC)
-#define DEFAULT_RATE_LIMIT_BURST 1000
+#define DEFAULT_RATE_LIMIT_INTERVAL (300*USEC_PER_SEC)
+#define DEFAULT_RATE_LIMIT_BURST 10000
```

Comment:
hmm, so this changes the rate limit from 1000 per 3s to 10,000 per 5min. Which besides being coarser doesn't actually change the bandwith we allow at all (i.e. both 1000/30 and 1000/30 is 33.3).

I think your intention is to actually increase the bandwidth here, no? Hence, just bump the BURST value, but leave the INTERVAL level the way it is?

Reply:
Hm, right, that's the intention...